### DISCUSSION TITLE
Suggestions for alternatives to AWS Amplify please

### DISCUSSION THREAD

**Discussion by DiegoMcDipster** [vercel/next.js#42858] [Help]
Hi All,

I've been working with Nextjs and AWS amplify for a couple of years now and seeing that Amplify only supports version 11 and Nextjs is now on version 13 I'm wondering what others are using as alternatives to Amplify.

So, I want to create an app that uses:
1. SSR (therefore Nextjs frontend)
2. Cognito, S3, API gateway to access lambda functions which carry out the operations on DynamoDB (completely serverless)
3. Cypress test runner
4. CICD (currently done by Amplify)
5. typescript

Amplify does make the whole process very easy, Auth, API and Storage. But my concern is that the Amplify team is only supporting a version of Nextjs that's two versions old.

What alternative setups do you guys have?

**Comment by PSoltes** [comment]
Vercel
